praw/models/reddit/rules.py

```diff
@@ -147,7 +147,9 @@ def __call__(self) -> Iterator:
             category=DeprecationWarning,
             stacklevel=2,
         )
-        return self.__iter__()
```

Comment:
How was there not a failing test for this? Should we re-add the old test?

Reply:
I deleted the old test when making the new ones. I do think that the current test probably iterates over it.